src/tasks/internFunctional.js

```diff
@@ -0,0 +1,63 @@
+
+
+module.exports = function(gulp, defaults, subtasks) {
+
+    var argv = require('yargs').argv;
+    var gutil = require('gulp-util');
+    var shell = require('gulp-shell');
+    var wait = require('gulp-wait');
+
+    var taskName = 'test:intern';
+    var command = '';
+
+    if (argv.local) 
+    {
+        command = './node_modules/.bin/intern-runner config=tests/functional/internLocal';
+        gulp.task(taskName, ['begin_Testing'], function (done) {
+            gulp.start(['connect:stop']);
+            done();
+        })
+    }
+    else if (argv.sauce) 
+    {
+        command = './node_modules/.bin/intern-runner config=tests/functional/internSauce';
+        gulp.task(taskName, ['connect:noReload','start_Test'], function (done) {
+            gulp.start(['connect:stop']);
+            done();
+        })
+    }
+    else 
+    {
+        gulp.task(taskName, function (done) {
+            gutil.log(gutil.colors.red("Need to pick a configuration. (--sauce or --local)"));
+            done();
+        })
+    }
+
+    gulp.task('start_Test', function () {
```

Comment:
since the connect:noReload is being referenced from the default task file should i still reference it as 'connect:noReload' when defining my task or should i assign it to a variable, connectNoReload = 'connect:noReload' and reference that variable?